charts/merlin/ci/ci-values.yaml

```diff
@@ -9,6 +9,8 @@ global:
     uiPrefix: "/"
     uiServiceName: mlp
 deployment:
+  annotations:
```

Comment:
did you mean to add this to unit tests?  i don't think it is needed for ci values.

Reply:
It's not needed but its a quick way to test the rendering of the annotations. Will remove it since it works and there's not much logic to test actually.